docs/usage/faq-troubleshooting.md

```diff
@@ -0,0 +1,9 @@
+# Troubleshooting and FAQs
```

Comment:
ok, so that was removed from the usage.md

Reply:
first pipeline I am seeing this, love the new structure on the website